packages/portal/src/components/generic/LinkGroup.vue

```diff
@@ -77,19 +75,12 @@
       },
 
       /**
-       * Optional class for styling of links
+       * Variant for specific styles
+       * @values light, social
```

Comment:
We always use the "light" variant. Let's just remove that and call it "default", then the only variant needs to be "social".